_api_app/app/Shared/Storage.php

```diff
@@ -282,6 +282,9 @@ protected function xmlFile2array($xml_file)
      */
     protected function copyFolder($src, $dst)
     {
+        if (!file_exists($this->XML_SITES_ROOT)){
+            mkdir($this->XML_SITES_ROOT);
+        }
         $dir = opendir($src);
         @mkdir($dst);
```

Comment:
I think we could change only this line like this, could you test it?
`@mkdir($dst, 0755, true)`